### PR TITLE
[IMP] mail: attachment preview on ir_attachment kanban

### PR DESCRIPTION
**Pull request by msh-odoo**
-- Open attachment preview on click of kanban
-- Image, video, PDF are supported for preview, other then this extension files will work normally

Task: https://www.odoo.com/web#id=37550&view_type=form&model=project.task&action=333&active_id=131&menu_id=4720
Pad: https://pad.odoo.com/p/r.a3b280bd04c5b6d6052e8809995e316c

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Comment by msh-odoo** [comment]
LGTM

File: addons/mail/static/src/js/ir_attachment_preview.js [hunk 1]
```diff
@@ -0,0 +1,59 @@
+odoo.define('mail.ir_attachment_preview', function (require) {
```
[the remark this answers is not in the document]
Comment (msh-odoo) [reply addons/mail/static/src/js/ir_attachment_preview.js]: done

**Comment by rdeodoo** [comment]
@ThanhDodeurOdoo According to the task you were going to handle this one.
I guess we can close this PR now?

**Comment by ThanhDodeurOdoo** [comment]
@rdeheele yes in v12, documents improves the views of ir.attachment, but since 12.1, it is its own records so the kanban view of ir.attachment doesn't have this feature.

Although I also added an attachment button in the chatter that shows all the attachments of the record (which replaced the attachment menu in the sidebar) so it's not necessary to have these previews anymore.

**Pull request closed without merging**